packages/vats/src/lib-board.js

```diff
@@ -1,22 +1,55 @@
 // @ts-check
+/**
+ * @file lib-board: share objects by way of plain-data ids
+ *
+ * @see prepareBoardKit()
+ */
 
-import { assert, details as X, q, Fail } from '@agoric/assert';
+import { assert, q, Fail } from '@agoric/assert';
 import { E, Far } from '@endo/far';
 import { makeMarshal } from '@endo/marshal';
-import { makeScalarMapStore } from '@agoric/store';
-import { crc6 } from './crc.js';
+import {
+  M,
+  makeScalarBigMapStore,
+  prepareExoClassKit,
+  provideDurableMapStore,
+} from '@agoric/vat-data';
+import { makeEphemeraProvider } from '@agoric/zoe/src/contractSupport/durability.js';
 
-import './types.js';
+import { crc6 } from './crc.js';
 
 export const DEFAULT_CRC_DIGITS = 2;
 export const DEFAULT_PREFIX = 'board0';
 
+// #region Interface Guards
+const CapDataShape = { body: M.string(), slots: M.array() };
+const MarshalI = M.interface('Marshaller', {
```

Comment:
you talked about this being a marshaller specific to Board. might want to name it that to convey that it not be refactored for other purposes